### ISSUE TITLE
1.14 backport?

### ISSUE THREAD

**Issue by mattmess1221** [killjoy1221/TabbyChat-2#110]
***tl;dr NO***

I do know that 1.14 is LTS for Forge, but I don't see myself being able to backport the changes I've done for 1.15. Most of them would be able to come over no problem since they don't depend on MC internals, but specific chat features (auto completion, suggestion box, etc) would be significantly harder.

As it sits right now, the 1.14 version of this mod was never stable enough for release. It also required many hacks, access transforms, and vanilla code duplication to get working. 1.15 is *so* much better for this because it properly separates all the features so I can reuse them. In short:

### I will not be supporting, releasing, or updating a 1.14 version of this mod.

If you have any issues, be sure to create them with 1.15 (or later). I'll be keeping this issue open and pinned for a while so it will be visible.
